tslint.json

```diff
@@ -0,0 +1,72 @@
+{
+  "rules": {
+    "class-name": false,
+    "comment-format": [
+      true
+    ],
+    "curly": false,
+    "eofline": false,
+    "forin": false,
+    "indent": [
+      true,
+      "spaces"
+    ],
+    "label-position": true,
+    "max-line-length": false,
+    "member-access": false,
+    "member-ordering": [
+      true,
+      "static-after-instance",
+      "variables-before-functions"
+    ],
+    "no-arg": true,
+    "no-bitwise": false,
+    "no-console": false,
+    "no-construct": true,
+    "no-debugger": true,
+    "no-duplicate-variable": true,
+    "no-empty": false,
+    "no-eval": true,
+    "no-inferrable-types": true,
+    "no-shadowed-variable": true,
+    "no-string-literal": false,
+    "no-switch-case-fall-through": true,
+    "no-trailing-whitespace": true,
+    "no-unused-expression": true,
+    "no-use-before-declare": false,
+    "no-var-keyword": true,
+    "object-literal-sort-keys": false,
+    "one-line": [
+      true,
+      "check-open-brace",
+      "check-catch",
+      "check-else",
+      "check-whitespace"
+    ],
+    "quotemark": false,
+    "radix": true,
+    "semicolon": [
+      "always"
+    ],
+    "triple-equals": [],
+    "typedef-whitespace": [
+      true,
+      {
+        "call-signature": "nospace",
+        "index-signature": "nospace",
+        "parameter": "nospace",
+        "property-declaration": "nospace",
+        "variable-declaration": "nospace"
+      }
+    ],
+    "variable-name": false,
+    "whitespace": [
+      true,
+      "check-branch",
+      "check-decl",
+      "check-operator",
+      "check-separator",
+      "check-type"
+    ]
+  }
+}
```

Comment:
Is there any way to at least not have these repeated between all our projects?

Reply:
Probably not.

Reply:
at least not yet, I think.